tests/sentry/tasks/test_daily_summary.py

```diff
@@ -0,0 +1,179 @@
+import copy
+from datetime import datetime, timedelta, timezone
+
+from sentry.constants import DataCategory
+from sentry.models.activity import Activity
+from sentry.models.group import GroupStatus
+from sentry.services.hybrid_cloud.user_option import user_option_service
+from sentry.tasks.summaries.daily_summary import prepare_summary_data
+from sentry.tasks.summaries.weekly_reports import ONE_DAY, OrganizationReportContext
+from sentry.testutils.cases import OutcomesSnubaTest, SnubaTestCase
+from sentry.testutils.factories import DEFAULT_EVENT_DATA
+from sentry.testutils.helpers.datetime import before_now, freeze_time, iso_format
+from sentry.testutils.silo import region_silo_test
+from sentry.types.activity import ActivityType
+from sentry.types.group import GroupSubStatus
+from sentry.utils.dates import to_timestamp
+from sentry.utils.outcomes import Outcome
+
+
+@region_silo_test
+class DailySummaryTest(OutcomesSnubaTest, SnubaTestCase):
+    def store_event_and_outcomes(
+        self, project_id, timestamp, fingerprint, category, num_times, release=None
+    ):
+        data = {
+            "event_id": "a" * 32,
+            "timestamp": iso_format(timestamp),
+            "stacktrace": copy.deepcopy(DEFAULT_EVENT_DATA["stacktrace"]),
+            "fingerprint": [fingerprint],
+        }
+        if release:
+            data["release"] = release
+        event = self.store_event(
+            data=data,
+            project_id=project_id,
+        )
+        self.store_outcomes(
+            {
+                "org_id": self.organization.id,
+                "project_id": project_id,
+                "outcome": Outcome.ACCEPTED,
+                "category": category,
+                "timestamp": timestamp,
+                "key_id": 1,
+            },
+            num_times=num_times,
+        )
+        group = event.group
+
+        group.status = GroupStatus.RESOLVED
+        group.substatus = None
+        group.resolved_at = timestamp + timedelta(minutes=1)
+        group.save()
+        return group
+
+    @freeze_time(before_now(days=2).replace(hour=12, minute=0, second=0, microsecond=0))
+    def test_prepare_summary_data(self):
+        now = datetime.now().replace(tzinfo=timezone.utc)
+        two_hours_ago = now - timedelta(hours=2)
+        two_days_ago = now - timedelta(days=2)
+        three_days_ago = now - timedelta(days=3)
+        self.project.first_event = three_days_ago
+        self.project.save()
+        project2 = self.create_project(
```

Comment:
Probably out of scope for MVP - but would be cool to have a setup function that creates the state for the test, then we could more easily have multiple test definitions for different test states.